spec/models/user_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'random_hash'
```

Comment:
If you can pull the require outside the `User` class definition, you might be able to remove this line.